apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -19,8 +41,92 @@
    */
   public static void main(String[] args) {
     LOGGER.info("Successfully started");
-    LOGGER.info("Hello, World!");
+
+    AppConfiguration appConfig = null;
+    try {
+      appConfig = AppConfiguration.readConfigFromEnvironmentVariables();
+      LOGGER.info("Application configured: '{}'", appConfig);
+    } catch (AppConfigurationException e) {
+      System.err.println(e.getMessage());
+      LOGGER.warn("Invalid app configuration.", e);
+      System.exit(EXIT_CODE_BAD_CONFIG);
+    }
+
+    MetricRegistry appMetrics = setupMetrics(appConfig);
+
+    // Create a data source for use by createOrUpdateSchema
+    HikariDataSource pooledDataSource =
+        createPooledDataSource(appConfig.getDatabaseOptions(), appMetrics);
+
+    // run migration
+    boolean migrationSuccess =
+        DatabaseSchemaManager.createOrUpdateSchema(pooledDataSource, appConfig);
+
     LOGGER.info("Shutting down");
-    System.exit(0);
+    if (!migrationSuccess) {
+      System.exit(EXIT_CODE_FAILED_MIGRATION);
+    }
+    System.exit(EXIT_CODE_SUCCESS);
+  }
+
+  /**
+   * Sets the metrics.
+   *
+   * @param appConfig the app config
+   * @return the metrics
+   */
+  private static MetricRegistry setupMetrics(AppConfiguration appConfig) {
```

Comment:
seems like it could be in bfd-shared-utils

Reply:
Added a TODO to handle this in 1558.